src/js/media-controller.js

```diff
@@ -880,6 +942,34 @@ const monitorForMediaStateReceivers = (
   return unsubscribe;
 };
 
+let testMediaEl;
+export const getTestMediaEl = (nodeName = 'video') => {
```

Comment:
The interaction between nodeName and testMediaEl could have unexpected results if you actually need an audio or video el to be returned and one was already created. I would just remove the nodeName option until we know we need it.

Reply:
Alternatively, make `testMediaEl` a Map keyed off of nodeName (but  likely yagni)

Reply:
Cool good call. Will remove for now.